core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/UserApiServiceSpec.scala

```diff
@@ -137,6 +139,8 @@ class UserApiServiceSpec extends ApiServiceSpec {
       sealRoute(services.userRoutes) ~>
       check {
         assertResult(StatusCodes.BadRequest)(status)
+        val responseString = Unmarshal(response.entity).to[String].futureValue
+        assert(responseString.contains("Project cannot be deleted because it contains workspaces."))
```

Comment:
This billing project does contain a v1 workspace, but it contains around 22 workspaces in total (mostly v2).